# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/moz.build
# Commit: 028af3d71e19
# Full Hash: 028af3d71e1918406a0bc3b249ef8feaa0348643
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-13 21:42:25
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Only scan wifi when the network changes or when on mobile r=necko-reviewers,kershaw
#   
#   Simplifies the concurrent operations of wifi scanning and reduces the
#   frequency in common cases.  Wifi scanning when on mobile is reduced from
#   every 5 seconds to every minute.  Wifi scans will also happen whenever
# ==============================================================================

diff -r 35c408b9531d -r 028af3d71e19 netwerk/wifi/moz.build
--- a/netwerk/wifi/moz.build	Sat May 13 00:36:00 2023 +0000
+++ b/netwerk/wifi/moz.build	Sat May 13 00:36:01 2023 +0000
@@ -56,6 +56,8 @@
     ]
     CXXFLAGS += CONFIG["MOZ_DBUS_GLIB_CFLAGS"]
 
+TEST_DIRS += ["gtest"]
+
 FINAL_LIBRARY = "xul"
 
 include("/ipc/chromium/chromium-config.mozbuild")